apps/src/templates/codeDocs/ProgrammingExpressionOverview.jsx

```diff
@@ -166,6 +166,13 @@ export default function ProgrammingExpressionOverview({
           />
         </div>
       )}
+      <div>
+        <p />
+        <p>
+          Found a bug in the documentation? Let us know at{' '}
```

Comment:
the linter told me to do this 😓 . without a space at the end of this line, there's no space before the link when it's rendered.

Reply:
Lol that kind of thing is so frustrating- I'm totally fine with this.

Reply:
My only other thought- we could add the space at the start of the href link instead🤷🏻‍♀️

Reply:
Can we add this string to `apps/i18n/common/en_us.json` so it can be translated?

Reply:
Oh good point, I'll do that